src/work-allocation/components/task-manager-filter/task-manager-filter.component.ts

```diff
@@ -266,6 +270,8 @@ export class TaskManagerFilterComponent implements OnInit, OnDestroy {
       TaskManagerFilterComponent.initRoleTypeFilter(),
       TaskManagerFilterComponent.findPersonFilter(),
       TaskManagerFilterComponent.initTaskTypeFilter(),
+      TaskManagerFilterComponent.initTaskNameFilter()
+
     ];
     if (this.isRelease4) {
```

Comment:
We should actually be using this - I forced the above in to check easier

Reply:
What do you mean?